PhpAmqpLib/Channel/AMQPChannel.php

```diff
@@ -342,6 +341,10 @@ public function exchange_declare(
         $arguments = null,
         $ticket = null
     ) {
+        if (!$this->is_open) {
+            throw new AMQPRuntimeException('Channel connection is closed.');
```

Comment:
Java, .NET, and Ruby (Bunny) clients says something like "an operation was attempted on a closed channel."

I guess in PHP connection and channel lifecycles are often inseparable, I recommend that the message focuses on channel being closed. It can be due to a channel-level protocol exception or connection closure or TCP connection loss: the end result for the caller is the same.